src/amp/lib/permutive.test.ts

```diff
@@ -0,0 +1,56 @@
+import { generatePermutivePayload } from './permutive';
+
+describe('generatePermutivePayload', () => {
+    test('generates the right payload given a complete config', () => {
+        const config = {
+            isPaidContent: false,
+            pageId: 'page id',
+            headline: 'article headline',
+            section: 'sport',
+            author: 'author name',
+            keywords: 'keyword1, keyword2',
```

Comment:
are the spaces going to matter here if they appear in the real data? I think Permutive is expecting a comma separated string, I don't know if they'll be stripping whitespace on what's provided.

Reply:
As far as I'm aware, both keywords and authors in the config store come as a comma separated string with no whitespace. But agree better to be explicit.